run/test/specs/input_validations/onboarding_long_name.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { androidIt, iosIt } from '../../../types/sessionIt';
+import * as onboarding from '../locators/onboarding';
+import { openAppOnPlatformSingleDevice, SupportedPlatformsType } from '../utils/open_app';
+import { localize } from '../../../localizer/i18n/localizedString';
+
+iosIt('Onboarding long name', onboardingLongName);
+androidIt('Onboarding long name', onboardingLongName);
+
+// the libSession limit for display names is 100 bytes but for this purpose 101 chars = 101 bytes will do    
+const tooLongName = 'One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in his bed int';
+if (tooLongName.length <= 100) {  
```

Comment:
Nice idea but unnecessary to test our own test inputs.

Reply:
My experience tells me that a check like this costs next to nothing and can save valuable time debugging failing tests. 
Of course, a diligent code reviewer catches an error like this such that the situation that I'm guarding against never happens in the first place. Happy to remove it.

Reply:
Yeah I agree, I think its better to err on the side of caution, so happy to leave it in. But I would just move it to inside the test for readability. When going through the error logs its nice to have the error be right next to the check.